models/object_detection/rtmdet/requirements.txt

```diff
@@ -1,7 +1,10 @@
-https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl
-https://download.openmmlab.com/mmcv/dist/cu118/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl
+#https://download.pytorch.org/whl/cu118/torch-2.0.0%2Bcu118-cp39-cp39-linux_x86_64.whl #select the version for your cuda version
+torch==2.0.0
+#https://download.openmmlab.com/mmcv/dist/cu124/torch2.0.0/mmcv-2.0.0-cp39-cp39-manylinux1_x86_64.whl #select the version for your cuda version
+numpy==1.23.5
 supervision @ git+https://github.com/roboflow/supervision.git@develop
 openmim==0.3.9
 mmdet==3.3.0
 mmengine==0.10.5
 mmyolo==0.6.0
+# after these intalls, you need to mim install mmcv==2.0.0
```

Comment:
The `rtmdet` directory contains a `README.md` file. Please update it to include all the steps needed to run the RTDet benchmark, step by step, just as you did previously.